File: official/docs/csharp/orders/OneCallBuyOrder.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Order order = await Order.Create(new Dictionary<string, object>() {
  { "carrier_accounts",
    new List<String>(){
        "ca_..."
  } },
  { "service", "NextDayAir" },
  { "to_address", new Dictionary<string, string>()
    { "id", "adr_..."}
  },
  { "from_address", new Dictionary<string, string>()
    { "id", "adr_..." }
  },
  { "shipments", new List<Dictionary<string, object>>(){
    {
        "parcel", new Dictionary<string, object>()
        {
            { "weight", 10.2 }
        }
     },
     {
        "parcel", new Dictionary<string, object>()
        {
            { "predefined_package", "FedExBox" },
            { "weight", 17.5 }
        }
    }
    } }
});

Console.WriteLine(JsonConvert.SerializeObject(order, Formatting.Indented));

File: official/docs/php/orders/OneCallBuyOrder.php
<?php

require_once("../lib/easypost.php");
\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$order = \EasyPost\Order::create(array(
    "carrier_accounts" => 'ca_...', 
    "service" => 'NextDayAir',
    "to_address" =>array("id" => "adr_..."),
    "from_address" => array("id" => "adr_..."),
    "shipments" => array(
        array(
            "parcel" => array(
                "weight" => 10.2
            )
        ),
        array(
            "parcel" => array(
                "predefined_package" => "FedExBox",
                "weight" => 17.5
            )
        ),
    ),
));

echo $order;
